src/common/composables/amcharts5/index.ts

```diff
@@ -0,0 +1,56 @@
+import type { ComputedRef } from 'vue';
+import {
+    computed, reactive, toRefs, watchEffect,
+} from 'vue';
+
+import * as am5 from '@amcharts/amcharts5';
+import type { Root } from '@amcharts/amcharts5';
+import am5themes_Animated from '@amcharts/amcharts5/themes/Animated';
+
+import { createDonutChart, createPieChart, createPieSeries } from '@/common/composables/amcharts5/pie-chart-helper';
+import type { ChartContext } from '@/common/composables/amcharts5/type';
+import { createXYDateChart, createXYLineSeries, createXYStackedColumnSeries } from '@/common/composables/amcharts5/xy-chart-helper';
+
+export const useAmcharts5 = (
+    chartContext: ComputedRef<ChartContext>,
```

Comment:
```suggestion
    chartContext: Ref<ChartContext>,
```

computed ref 말고 그냥 ref 로 하면 어떨까요~?